loadtesting/k6/tests/sendErc20.test.js

```diff
@@ -0,0 +1,65 @@
+import { ethClient, sendErc20ViaTransferFunction } from './utils/ethClient.js';
+import { randomItem } from 'https://jslib.k6.io/k6-utils/1.2.0/index.js';
+import { erc20Address, transferAmountRange } from './utils/consts.js';
+import { sendTokenOptions } from '../options/options.js';
+import { Trend, Counter } from 'k6/metrics';
+import { check } from 'k6';
+import { SharedArray } from 'k6/data';
+import exec from 'k6/execution';
+
+const sendErc20Requests = new Counter('send_erc20_requests');
+const sendErc20ErrorCounter = new Counter('send_erc20_errors');
+const sendErc20ErrorReceiptStatusCounter = new Counter('send_erc20_errors_in_receipt');
+const sendErc20RequestTime = new Trend('send_erc20_request_time', true);
+
+export const options = sendTokenOptions;
+
+const pathToContractData = '../contracts/ERC20/ERC20';
+const abi = JSON.parse(open(pathToContractData + '.abi'));
+
+const usersArray = new SharedArray('Users accounts', function () {
+    const accounts = JSON.parse(open("../data/accounts.json"));
+    let data = [];
+    for (let i = 0; i < Object.keys(accounts).length; i++) {
+        data[i] = accounts[i];
+    }
+    return data;
+});
+
+export default function sendErc20Test() {
+    const vuID = exec.vu.idInTest
+    const index = vuID % usersArray.length;
+
+    const accountSenderAddress = usersArray[index].sender_address;
+    const accountSenderPrivateKey = usersArray[index].sender_key;
+    const accountReceiverAddress = usersArray[index].receiver_address;
+    const client = ethClient(accountSenderPrivateKey);
+    const erc20 = client.newContract(erc20Address, JSON.stringify(abi), accountSenderPrivateKey)
+
+    const startTime = new Date();
+
+    try {
+        const receipt = sendErc20ViaTransferFunction(
+            client,
+            erc20,
+            accountSenderAddress,
+            erc20Address,
+            JSON.stringify(abi),
```

Comment:
you can define abi inside the fuction because it can't be different for this function

Reply:
We use open() function to read the .abi file. This function can be used only at the init stage. Unfortunately we can't move it to ethClient.js file and we can't use smth similar cause we do not have nodejs here